pandas/tests/series/test_operators.py

```diff
@@ -667,83 +667,188 @@ def test_div(self):
             assert_series_equal(result, expected)
 
 
-class TestTimedeltaSeriesArithmetic(object):
-    def test_timedelta_series_ops(self):
-        # GH11925
-        s = Series(timedelta_range('1 day', periods=3))
-        ts = Timestamp('2012-01-01')
-        expected = Series(date_range('2012-01-02', periods=3))
-        assert_series_equal(ts + s, expected)
-        assert_series_equal(s + ts, expected)
+class TestTimedeltaSeriesArithmeticWithIntegers(object):
+    # Tests for Series with dtype 'timedelta64[ns]' arithmetic operations
+    # with integer and int-like others
 
-        expected2 = Series(date_range('2011-12-31', periods=3, freq='-1D'))
-        assert_series_equal(ts - s, expected2)
-        assert_series_equal(ts + (-s), expected2)
+    # ------------------------------------------------------------------
+    # Addition and Subtraction
+
+    def test_td64series_add_int_series_invalid(self):
+        tdser = Series(['59 Days', '59 Days', 'NaT'], dtype='timedelta64[ns]')
```

Comment:
since you are repeating this everywhere, use a fixture for tdser

Reply:
Any wiggle room on this request?  I really prefer to be able to tell what is going on in a test just by looking at it, to be able to just copy/paste the text of a test when something goes wrong.

Reply:
no wiggle room. This just adds so much boilerplate. that is the entire purpose of fixtures, to avoid copy-paste boilerplate.